aws/resource_aws_security_group_rule_test.go

```diff
@@ -2089,6 +2103,7 @@ resource "aws_security_group_rule" "allow_self" {
   from_port                = 0
   to_port                  = 0
   protocol                 = "-1"
+  description              = "some description"
```

Comment:
Nit: It is generally preferred to create a new test and test configuration to ensure we are not introducing a regression. 👍